pkg/commands/overrides_exporter.go

```diff
@@ -108,9 +107,6 @@ func (o *OverridesExporterCommand) updatePresets(presetsMap map[string]*validati
 		o.presetsGauge.WithLabelValues(
 			"max_series_per_query", preset,
 		).Set(float64(limits.MaxSeriesPerQuery))
-		o.presetsGauge.WithLabelValues(
-			"max_samples_per_query", preset,
-		).Set(float64(limits.MaxSamplesPerQuery))
```

Comment:
Why was this removed?

Reply:
This was removed in cortex while removing support for chunk storage (https://github.com/cortexproject/cortex/pull/4812/files#diff-1e0a07c0c1ea87ab5a582d79c587a73a9b8e0680d89c61d5c94be106823a4a4eL61) 
check `limits.go`